bee-inx/src/raw.rs

```diff
@@ -44,19 +44,63 @@ impl From<proto::RawOutput> for Raw<bee::output::Output> {
     }
 }
 
+impl From<Raw<bee::output::Output>> for proto::RawOutput {
+    fn from(value: Raw<bee::output::Output>) -> Self {
+        Self { data: value.data }
+    }
+}
+
+impl From<proto::RawBlock> for Raw<bee::Block> {
+    fn from(value: proto::RawBlock) -> Self {
+        value.data.into()
+    }
+}
+
+impl From<Raw<bee::Block>> for proto::RawBlock {
+    fn from(value: Raw<bee::Block>) -> Self {
+        Self { data: value.data }
+    }
+}
+
+impl From<proto::RawMilestone> for Raw<bee::payload::milestone::MilestonePayload> {
+    fn from(value: proto::RawMilestone) -> Self {
+        value.data.into()
+    }
+}
+
+impl From<Raw<bee::payload::milestone::MilestonePayload>> for proto::RawMilestone {
+    fn from(value: Raw<bee::payload::milestone::MilestonePayload>) -> Self {
+        Self { data: value.data }
+    }
+}
+
 #[cfg(test)]
 mod test {
     use bee::rand::output::rand_output;
 
     use super::*;
+    use crate::ProtocolParameters;
 
     #[test]
     fn raw_output() {
-        let output = rand_output();
+        let protocol_parameters = bee::protocol::ProtocolParameters::default();
```

Comment:
You should probably use the test function `protocol_parameters` in the protocol module. Not sure the default values would be fine to build all bee-block objects.